apps/kserve/kserve/kustomization.yaml

```diff
@@ -1,6 +1,16 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  # Install Kserve in kubeflow namespace
-  - kserve_kubeflow.yaml
-  - kserve-cluster-resources.yaml
+# Install Kserve in kubeflow namespace
+- kserve_kubeflow.yaml
+- kserve-cluster-resources.yaml
+
+# Patch to delete the kserve-localmodelnode-agent DaemonSet
```

Comment:
why is it deleted? Is there some replacement for it?

Reply:
Because it lets you exploit the infrastructure. kserve also acknowledged it in https://github.com/kserve/kserve/issues/4402

Reply:
Thanks